CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 1.10.4 - 2024-07-08
+
+### Added
+
+- `Swift` terminal profile
+- Set Swift environment variables in the integrated terminal
+- Show macro expansions
```

Comment:
I think we should also add a bit of documentation to [settings.md](https://github.com/swiftlang/vscode-swift/blob/main/docs/settings.md) about how to enable macro expansion w/ the LSP arguments setting:

```
"swift.sourcekit-lsp.serverArguments": [
  "--experimental-feature", "show-macro-expansions"
]
```